src/lib/helpers/latestVersion.js

```diff
@@ -1,10 +1,28 @@
 'use strict'
 
-const packageJson = require('package-json')
+const { request } = require('undici')
 
-const lastestVersion = async (packageName, options) => {
-  const { version } = await packageJson(packageName.toLowerCase(), options)
-  return version
+/**
+ * Returns the latest version of a package under a certain tag
+ * @param {string} packageName
+ * @param {string} tag
+ * @returns {Promise<string>}
+ */
+const latestVersion = async (packageName, tag) => {
+  const response = await request(`https://registry.npmjs.org/${packageName}/${tag}`, {
+    method: 'GET',
+    headers: {
+      'Accept': 'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*',
+      'Content-Type': 'application/json'
+    }
+  })
+
+  if (response.statusCode !== 200) {
```

Comment:
i think ideally if the request fails in anyway, we would just return the current running version - so that the user does not receive an update notification.